test/index.js

```diff
@@ -26,8 +26,8 @@ setTimeout(_ => {
   assert(!result.includes('Error'), 'Expected Error line not to be included')
   const stacks = result.join()
   assert(!stacks.includes('async_hooks.js'), 'Expected async_hooks.js traces not to be included')
+  assert(!stacks.includes('async_hooks'), 'Expected async_hooks traces not to be included')
   assert(stacks.includes('test' + sep + 'index.js:18'), 'Expected test' + sep + 'index.js:18 in stack trace')
-  assert(stacks.includes('Server.connectionListener'), 'Expected Server.connectionListener in stack trace')
```

Comment:
Is there no useful information about going through the server implementation in the stacktrace anymore?

Reply:
Seems there are only internal timeouts mentioned in stacktrace for this test case.

Reply:
Oh that's sad. It'd be useful to the end user.